base/path.jl

```diff
@@ -196,6 +197,32 @@ function pathsep(paths::AbstractString...)
     return path_separator
 end
 
+"""
+    splitpath(path::AbstractString) -> Vector{String}
+
+Split a file path into all its path components. This is the opposite of
+`joinpath`. Returns an array of substrings, one for each directory or file in
+the path, including the root directory if present.
+
+# Examples
+```jldoctest
+julia> splitpath("/home/myuser/example.jl")
+("/", "home", "myuser", "example.jl")
```

Comment:
This output is going to be wrong now that this returns an array.

Reply:
👍 thanks! Fixed.